mocks/generate/tweets.ts

```diff
@@ -0,0 +1,39 @@
+import 'dotenv/config'
+import path from 'path'
+import {fileURLToPath} from 'url'
+import fs from 'fs'
+import {installGlobals} from '@remix-run/node'
+// NOTE: run this with tsx ./mocks/generate/tweets
+import {getTweet} from '../../app/utils/twitter/get-tweet.ts'
```

Comment:
the old code import `twitter.server` that was not exist
`const {getTweet} = require('../../app/utils/twitter.server')`